### ISSUE TITLE
Blur stream video behind semi-transparent chat background

### ISSUE THREAD

**Issue by Munzu** [akhanubis/twitch_chat_overlay_issues#8]
The chat can be hard to read if the background is (semi-)transparent. 
I'd love a feature where the chat background blurs the stream video behind the chat.

**Comment by akhanubis** [comment]
Something like this?

![image](https://user-images.githubusercontent.com/2922430/87859276-47d1a900-c90a-11ea-8f1e-098777cbd2f3.png)

**Comment by Munzu** [comment]
Yes exactly. I'd like to use something like that in conjunction with a background color.

**Comment by akhanubis** [comment]
I did some research and, unfortunately, there is no way to accomplish this on Firefox (backdrop-filter: blur not working with stream as background even when enabling gfx.webrender.all and layout.css.backdrop-filter.enabled).
It can be done in Chrome and Brave tho, so let me know if you are willing to switch to any of those browsers and I'll get to it.

**Comment by Munzu** [comment]
That's too bad. I'm okay staying on Firefox. But thank you so much!
